src/pydvl/value/stopping.py

```diff
@@ -623,3 +625,74 @@ def reset(self):
 
     def __str__(self):
         return f"HistoryDeviation(n_steps={self.n_steps}, rtol={self.rtol})"
+
+
+class RankStability(StoppingCriterion):
+    r"""A check for stability of Spearman correlation between checks.
+
+    When the change in rank correlation between two successive iterations is
+    below a given threshold, the computation is terminated.
+    The criterion computes the Spearman correlation between two successive iterations.
+    The Spearman correlation uses the ordering indices of the given values and correlates them.
+    This means it focuses on the order of the elements instead of their exact values.
+    If the order stops changing (meaning the Banzhaf semivalues estimates converge), the criterion stops the algorithm.
+
+    This criterion is used in :footcite:t:`wang_data_2022`.
```

Comment:
`footcite` doesn't work with mkdocs. Look at [oob.py](https://github.com/aai-institute/pyDVL/blob/develop/src/pydvl/value/oob/oob.py) for an example of how we do it currently.